### PR TITLE
Week labels in Timeline

### PR DESCRIPTION
**Pull request by jessmartin**
[Trello card](https://trello.com/c/2Lqou8ev/1-as-a-user-i-want-to-view-my-albums-grouped-by-week-of-release)

**Todo:**

- [ ] Refactor the `get_recent_albums_for` method to be a little cleaner

### CODE REVIEW

**Comment by jessmartin** [comment]
[As a user, I want to view my albums grouped by week of release](https://trello.com/c/2Lqou8ev/1-as-a-user-i-want-to-view-my-albums-grouped-by-week-of-release)

File: app/services/spotify_service.rb [hunk 1]
```diff
@@ -29,26 +29,34 @@ def self.authenticate(code)
   end
 
   def self.get_recent_albums_for(user:, released_after:)
```
Comment (rholdy) [inline app/services/spotify_service.rb]: Lets rename this variable to be more descriptive. released_after sounds like a boolean

File: spec/services/spotify_service_spec.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@
   end
 
   describe ".get_recent_albums_for" do
-    let(:user) { User.new(access_token: "BQCglneaf-osQ50FKcfsV0HRbABrfdF2PhZ9diu5LfFQC9canNvukI-xcMYcaGp5rjsC4NaMblc21d7CvI5YohxLLfKBPeLVwGDTtT5693-PEcyuMQXZfYX6shPy3c-bXgOy4y9m7VG--jZgNCopIhkX09x4Rqb9ECjdXqo4FPn52oaqPAlYB8WnkrBmHwdgQ6iV5amiz8x-MMaLMuo36-HsdOQI95vS4WO2CfCnQ3gf-FKTQrlX1SXnwNeFuvnV1DJYeEPVAJgFBcPZOrS712zuQmWJ-Pk") }
+    let(:user) { User.new(access_token: "BQBQPOPfqMewO4GN65W8PYV_fB-CLecFbkd_9xNdkm7m1ScR24gU8TMccz2Ld0ePQD1HX-umnvrfk7IYPNF-HrGpeaosC6eQlai9OSF5XFmtzqt9Rb1ni6jhZ7XsAfTTMkHV3xvK4tiYkPtB07nWxGvdo7aRllhYZ8LtDV3Whnwo-3YzvFhXz1fj3JhBvJeOgBURmS0HMT8GvLuaJH1tWnGsXYwfwt24fyHvmA6YTJ0-lCfFQrGNl3_jITDJNhetYkd9nBNl_970iqV7h2ZDdbqFGrcGr5cGqQ") }
```
Comment (rholdy) [inline spec/services/spotify_service_spec.rb]: Lets kill the token.

File: app/views/releases/index.html.erb [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+  <% current_year_and_week_number = "none" %>
+
   <% @albums.each do |album| %>
+    <!-- Does this album fall into a new week? -->
+    <% album_release_date = Date.parse(album["release_date"]).to_date %>
+    <% release_year_week = "#{album_release_date.year} Week #{album_release_date.cweek}" %>
+    <% if current_year_and_week_number != release_year_week  %>
+      <% current_year_and_week_number = release_year_week %>
+      <h2 class="week-label"><%= current_year_and_week_number %></h2>
+    <% end %>
```
Comment (rholdy) [inline app/views/releases/index.html.erb:13]: WTF? Lets move this to the controller and test it

**Review by rholdy — CHANGES_REQUESTED** [review]

**Comment by jessmartin** [comment]
Wow, Hound CI back again! I've now disabled Hound, so these annoying comments won't show up anymore.

File: spec/services/spotify_service_spec.rb (hunk 2 above)
Comment (jessmartin) [reply spec/services/spotify_service_spec.rb]: Yep! Nuked it and a few other things as well.

File: app/views/releases/index.html.erb (hunk 3 above)
Comment (jessmartin) [reply app/views/releases/index.html.erb:13]: Naw boooooiii. We'll make this better once we get it into production and get people starting to use it.

Additionally, we're gonna spend some time re-designing the front end very send, so this code will soon get refactored or go away entirely.

**Comment by jessmartin** [comment]
Verified working on localhost. Ready to merge!

**Pull request merged**